tgui/packages/tgui/interfaces/Apc.js

```diff
@@ -196,6 +197,7 @@ const ApcContent = (props, context) => {
             label="Emergency Lighting"
             buttons={(
               <Button
+                tooltip="Lights use internal power cell when no power available."
```

Comment:
when no power "is" available

Reply:
Yeah, I just wanted it to be shorter. Two extra words are not a problem.